psi4/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.19...4.0)
 cmake_policy(SET CMP0094 NEW)
+if (POLICY CMP0167)
+    # L2 not prepared for BoostConfig instead of FindBoost
+    cmake_policy(SET CMP0167 OLD)
```

Comment:
Is there anything that needs to be done on the EFV end?

Reply:
Hard to say, as the last time I looked at L2 deps was evaleev/libint#314 (https://github.com/loriab/libint/compare/configconfig...loriab:libint:l2deps). That mentions updating from ancient Boost (1.29) and reorganizing some of EFV's special checks on the install. From https://stackoverflow.com/a/79147222 , it sounds like requiring Boost 1.70 would be necessary and probably some `find_package(Boost CONFIG)` conversions.

It's not a big concern for me.